tests/irepela/homework_10/test_context_manager_try_except.py

```diff
@@ -0,0 +1,12 @@
+from irepela.homework_10.context_manager_try_except import Catcher
+
+
+def test_try_except():
+    with Catcher():
+        5 / 0
+
+    with Catcher(ZeroDivisionError):
+        5 / 0
+
+    with Catcher(TypeError, ZeroDivisionError):
+        5 / 0
```

Comment:
check negative case where you do not catch error

Reply:
Added case without exception